lib/rack/attack/base_proxy.rb

```
@@ -5,6 +5,43 @@
 module Rack
   class Attack
     class BaseProxy < SimpleDelegator
+      attr_reader :bypass_all_store_errors, :bypassable_store_errors
+
+      def initialize(store, bypass_all_store_errors: false, bypassable_store_errors: [])
```

wondering if it's better to have just one variable: `bypassable_store_errors` which can either by a `Symbol` (:all, :none), or an `Array`? Just to avoid the possible combinations of both variable sets:
- bypass_all_store_errors: false, bypassable_store_errors: [...]
- bypass_all_store_errors: true, bypassable_store_errors: []
- bypass_all_store_errors: false, bypassable_store_errors: [...]
- bypass_all_store_errors: true, bypassable_store_errors: []

---

I think that's a good approach. We could also raise a configuration error if both variables are defined